src/test/java/seedu/address/testutil/PersonUtil.java

```diff
@@ -22,7 +22,7 @@ public class PersonUtil {
      * Returns an add command string for adding the {@code person}.
      */
     public static String getAddCommand(Person person) {
-        return AddCommand.COMMAND_WORD + " " + getPersonDetails(person);
+        return AddPersonCommand.COMMAND_WORD + " " + getPersonDetails(person);
     }
 
     /**
```

Comment:
For some of the constructors like Pet we need to change the old implementation later, just merge first we can fix the bugs later, good job!